src/components/BooksRender.jsx

```diff
@@ -1,11 +1,24 @@
 import BooksForm from './BooksForm';
 import BooksList from './BooksList';
+import imagete from '../imagete.png';
 
 const BooksRender = () => (
   <>
-    <BooksList />
-    <div className="Line" />
-    <BooksForm />
+    <div className="panel-bg">
+      <div className="bar-panel-bg">
+        <div>
+          <span><h2 className="titre">Bookstore CMS</h2></span>
+          <span className="books">BOOKS</span>
+          <span className="categorie">CATEGORIES</span>
+        </div>
+        <div>
+          <img src={imagete} alt="person" className="Oval-tete" />
+        </div>
+      </div>
+      <BooksList />
```

Comment:
good job on removing the scroll on the webpage but that doesn't solve the problem of the whole content not fitting within my browser interface. please fix this and try again 

![image](https://github.com/francksefu/booksmr/assets/60612329/ebb492bb-da87-4966-bbd6-0b5f53c0aa71)